src/Queue/src/Bootloader/QueueBootloader.php

```diff
@@ -55,6 +55,10 @@ public function init(Container $container, EnvironmentInterface $env, AbstractKe
             foreach ($config->getRegistryHandlers() as $jobType => $handler) {
                 $registry->setHandler($jobType, $handler);
             }
+
+            foreach ($config->getRegistrySerializers() as $jobType => $format) {
+                $registry->setSerializerFormat($jobType, $format);
```

Comment:
developers should have an ability to set not only format, but custom serializers also